src/dtos/group.dto.ts

```diff
@@ -0,0 +1,6 @@
+import { IsString } from 'class-validator';
+
+export class CreateGroupDto {
+  @IsString()
```

Comment:
if we not handling the Group name length in the front end, we can add it here as well. Otherwise, it looks good to me.